src/framework/audio/internal/synthesizers/synthesizersregister.cpp

```diff
@@ -51,6 +54,11 @@ std::vector<std::shared_ptr<ISynthesizer> > SynthesizersRegister::synthesizers()
     return synths;
 }
 
+async::Channel<ISynthesizerPtr> SynthesizersRegister::synthesizerAddedNotify() const
+{
+    return m_synthNotificationChannel;
```

Comment:
I think the `Notify` in the method name is superfluous, and it is better for the channel name to match the name of the method, i.e.
```
async::Channel<ISynthesizerPtr> SynthesizersRegister::synthesizerAdded() const
{
    return m_synthesizerAdded;
}
```